src/bot.coffee

```diff
@@ -1,6 +1,8 @@
-{ Adapter, TextMessage, EnterMessage, LeaveMessage, TopicMessage, CatchAllMessage, Robot } = require.main.require 'hubot'
+{ Adapter, TextMessage, EnterMessage, LeaveMessage, TopicMessage, CatchAllMessage } = require.main.require 'hubot'
 { SlackTextMessage, ReactionMessage } = require './message';
+{ Robot } = require './extensions';
```

Comment:
the `extensions.coffee` file doesn't have any exports, so this seems incorrect ( `Robot` will just be set to `undefined`). `Robot` isn't really used directly anywhere in this file, so i guess its safe to delete this line. the only side effect was that requiring this file extended the `Robot` class with the `react()` prototype method. We get that from line 2 of `slack.coffee`. If load order is an issue (i'm honestly not sure) then maybe that should be moved to line 1 instead.